src/policy/fees.h

```diff
@@ -50,6 +50,8 @@ enum class FeeEstimateMode {
     UNSET,        //!< Use default settings based on other criteria
     ECONOMICAL,   //!< Force estimateSmartFee to use non-conservative estimates
     CONSERVATIVE, //!< Force estimateSmartFee to use conservative estimates
+    BTC_KB,       //!< Use explicit BTC/kB fee given in coin control
+    SAT_B,        //!< Use explicit SAT/B fee given in coin control
```

Comment:
It seems pretty ugly to pass the various units into fee estimate mode here. Can't we convert to EXPLICIT for internal stuff?

Reply:
I don't understand what you are suggesting.

Reply:
BTC/kB and sat/B should be limited to the RPC interface. Once it gets to CCoinControl, it should just be EXPLICIT.

Reply:
Ahh, but that complicates the parser part though. Right now, it can take a string and output a `FeeEstimateMode`. With your suggestion that wouldn't be sufficient.

Reply:
It may actually simplify it. There's only a single call that can be inlined here.